lib/angular-payments.js

```diff
@@ -135,7 +126,16 @@ angular.module('angularPayments', []);angular.module('angularPayments')
       length: [16, 17, 18, 19],
       cvcLength: [0, 3],
       luhn: true
-    }
+    }, {
+      type: 'maestro',
+      brand: 'maestro',
+      pattern: /^(5018|5020|5038|6)/,
```

Comment:
Pattern updated according wiki: https://en.wikipedia.org/wiki/Payment_card_number#Issuer_identification_number_.28IIN.29

Reply:
You tested it in staging?

Reply:
yes, https://delphinus.transferwise.com/

Reply:
👌

Reply:
https://support.worldpay.com/support/kb/bg/testandgolive/tgl5103.html